### PR TITLE
Add quarkus to V0.0.2 branch

### PR DESCRIPTION
**Pull request by lance**
The tutorial doesn't quite work without this.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,25 +1,46 @@
-/target/
+<<<<<<< HEAD
```
Comment (Ladicek) [inline .gitignore]: Not really :-)

Comment (lance) [reply .gitignore]: OMG - I was worried about this. Fixing...

So, not really a good excuse but this was confusing to me until I realized that I simply had not CTL-s the file in VSCode. So the file looked right in my editor, but had not been saved on disk.

File: .gitignore [hunk 2]
```diff
@@ -1,25 +1,31 @@
...
-*.iml
-*.ipr
+docker-build.yaml
 
-### NetBeans ###
-/nbproject/private/
-/build/
-/nbbuild/
-/dist/
-/nbdist/
```
Comment (kameshsampath) [inline .gitignore:24]: @lance - don’t remove all these, I have settings for all common IDEs that a developer uses

**Review by kameshsampath — CHANGES_REQUESTED** [review]
Btw am not using v0.0.2 in updated instructions. Its better I remove this branch

File: .gitignore (hunk 2 above)
Comment (lance) [reply .gitignore:24]: @kameshsampath I simply rebased from `master`. Those entries do not exist there: https://github.com/redhat-developer-demos/knative-tutorial-greeter/blob/master/.gitignore

**Comment by lance** [comment]
> Btw am not using v0.0.2 in updated instructions. Its better I remove this branch

Do you think that it would be good to have versioned releases of the tutorial? I'm finding it hard to keep up with a moving target while trying to follow the tutorial.

**Comment by kameshsampath** [comment]
Yes I will release v0.4.0 ( just to keep inline with Knative release) early next week. This release was a big and hence having it snapshot for quite sometime to get feedback and review. Am also finding it hard to keep up
with Knative updates 😬

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by kameshsampath** [comment]
@lance - i dont have refs to v0.0.2 in tutorial wondering if we need to keep v0.0.2 branch. I am planning  to delete it. thoughts?

**Pull request closed without merging**